game-core/src/main/java/games/strategy/triplea/ai/weak/WeakAi.java

```diff
@@ -2,6 +2,7 @@
 
 import com.google.common.collect.Streams;
```

Comment:
Side-note: `WeakAI` we want to delete. The only reason we do not delete is for battle simulation. We new up a `WeakAI` to provide logic for the battle simulation. If we can extract that individual logic and supply it to battle simulation / battle calc, then we could remove this AI class entirely.

Reply:
Ack.